### ISSUE TITLE
IncSearch is GREAT + quick question

### ISSUE THREAD

**Issue by sfd99** [onozaty/browser-online-bookmark-incsearch#8]
Thanks for **IncSearch**!.
Really useful.

**Q1)**  Problem:
Each time I restart the Chrome browser,
**IncSearch** forgets my previous Pinboard db **IncSearch**.

So, 
I have to re-index all my Pinboard bookmarks db _again_
inside **IncSearch**,  every day...

Is there a way for **IncSearch**
to "remember" my previously indexed Pinboard db?.
(so I don't need to re-index 
the **IncSearch** Pinboard db
each day again, when I restart Chrome?)

**Q2)**   Anyhow, 
where is this **IncSearch** Pinboard db file 
stored  / saved
in my Ubuntu Linux PC?.
(I could not find this db file...).

Thanks!
**SFd99
San Francisco
Ubuntu LINUX 20.04
Chrome browser Version 93.0.4577.82**
Online Bookmark Incremental Search Version 2.0.2

**Comment by onozaty** [comment]
The db will be saved as a configuration for the extension.

Referring to the following site, 

* https://chromium.googlesource.com/chromium/src/+/HEAD/docs/user_data_dir.md#linux

It seems to be placed in the following location on Linux.

```
~/.config/google-chrome/Default/Local Extension Settings/lpaleambpedcnbeaolhoandbffjgidao
```

It's a problem if it reset on reboot, so I will prepare Ubuntu to check it out.

Thanks.

**Comment by sfd99** [comment]
**_Thank you, Onozaty!._**

Looking forward 
to your technical conclusions / opinion,
on why this is happening
on every  restart of Chrome.

**IncSearch** is such a wonderful
and useful program for Chrome & Pinboard.

Will stay tuned.
Hope you can find the reason and solution.

_Domo / Thanks,_

**SFd99
San Francisco
Ubuntu LINUX 20.04
Chrome browser Version 93.0.4577.82
Online Bookmark Incremental Search Version 2.0.2**

**Comment by onozaty** [comment]
I tried the following, and the bookmarks were still loaded after reboot.

* Ubuntu 20.04.3 LTS
* Chrome Version 94.0.4606.61
* Online Bookmark Incremental Search 2.0.2

If you have not tried it, could you please try reinstalling the extension?

Thanks

**Comment by sfd99** [comment]
Hi Onozaty,

I made a 
100% fresh, new Install of the Chrome Browser
and the IncSearch extension.
(it is now _the only_ extension installed...).

So now my config
is the same as yours:
   - Ubuntu 20.04.3 LTS
   - Chrome Version 94.0.4606.61
   - Online Bookmark Incremental Search 2.0.2

After login and sync with **Pinboard**,
IncSearch always works and searches GREAT!.

_But.._.
if I close and _then re-open_ the Chrome Browser,
it will request to again **Sync** with Pinboard,
(which requires the Pinboard ID and password, again).

So, it seems to "forget" 
the Pinboard login status 
and all the sync bookmarks file,
if the browser is simply  **closed** 
_or_ even after a **reboot**.

Is there a way (**cookie**, etc...).
to make IncSearch "remember"
the previously synched Pinboard bookmaks?.

btw: 
 Same problem with the Chromium browser...

Mystery!  :-)
**SFd99
San Francisco
Ubuntu LINUX 20.04
latest Chrome and Chromium browsers
Online Bookmark Incremental Search 2.0.2**

**Comment by onozaty** [comment]
In my environment, bookmarks are saved.

![ubuntu](https://user-images.githubusercontent.com/47337/134794222-93fe162b-75b9-4f5a-aaaf-1688c6b9b554.gif)

It is possible that there is some kind of error and your bookmarks are not being saved.
Can you please check if there is an error in Chrome's DevTools?

**Comment by sfd99** [comment]
**_Hi Onozaty,_**

Thanks for the clear short video.

I'm not a deeply Tech person,
so not familiar with Chrome DevTools, 
( I am more of an "End User type" ).

_But..._
I pressed [ F12 ] in Chromium browser
as you requested.

Yes, there is an Error Message!,
(snapshot image below):

![IncSearch_DevTools_Error](https://user-images.githubusercontent.com/3316239/134818946-cb925ca2-04ee-4647-a585-6358febf569a.png)

The red X Error mentions:
   **"QUOTA_BYTES quota EXCEEDED in index.html"**
where:
**index.html**  link is:
~~~
   chrome-extension://lpaleambpedcnbeaolhoandbffjgidao/ui/index.html
~~~
_Hmmm..._
maybe this quota error is because
I have almost **40 thousand (private) bookmarks**
in **Pinboard**?.

That's _a lot of bookmarks in **Pinboard**_.
This is why IncSearch 
is for me, such a fantastic, invaluable tool 
to search so many bookmarks in **Pinboard**!!!.

Onozaty, 
do you think
this may be the reason
for IncSearch loading ALL Bookmarks 
and searching 100% OK,
but later "forgetting" the bookmarks
when I restart Chrome?...

**SFd99
San Francisco
Ubuntu LINUX 20.04
latest Chrome and Chromium browsers
Online Bookmark Incremental Search 2.0.2**

**Comment by onozaty** [comment]
Thank you for the information.
I'd like to fix this by removing the size limitation.

Please wait for a while.

**Comment by onozaty** [comment]
It has been released as version 2.0.3, so could you please try it out?

**Comment by sfd99** [comment]
****IncSearch** version 2.0.3**
**now working GREAT 100%**,
_even after_ a  browser restart!.
(I tested in _both_, Chromium and Chrome browsers).

All PINBOARD bookmarks 
are remembered and displayed fine now,
without need to re-login and sync again
every time.

You fixed it.
_YAY!!_

Domo / Thanks 10^6, Onozaty.
You are a _real_ professional.

**SFd99
San Francisco
Ubuntu LINUX 20.04
latest Chrome and Chromium browsers
Online Bookmark Incremental Search 2.0.3**
